src/compiler/transformers/module/module.ts

```diff
@@ -1813,21 +1836,61 @@ namespace ts {
         }
     }
 
+    export const createBindingHelper: UnscopedEmitHelper = {

```

Comment:
Do you need to add these to getAllUnscopedEmitHelpers so that when we path output with prepends (with --out) in project references it doesn't duplicate these..

Reply:
That’s&nbsp;from&nbsp;<https://github.com/microsoft/TypeScript/pull/35967>.

This&nbsp;PR&nbsp;will only&nbsp;be&nbsp;<https://github.com/microsoft/TypeScript/commit/92aeea326b1796931945a6cd9ff9e78ba3e41ec2> once&nbsp;<https://github.com/microsoft/TypeScript/pull/35967> is&nbsp;merged.

Reply:
This is in `master` as of 01f81df